README.md

```diff
@@ -21,10 +21,10 @@
-**dev16.0-preview2**|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=dev16.0-preview2&jobname=Windows_VisualStudio_Integration_Tests&configuration=debug)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=dev16.0-preview2&view=logs)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=dev16.0-preview2&jobname=Windows_VisualStudio_Integration_Tests&configuration=release)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=dev16.0-preview2&view=logs)|
-**dev16.0-preview2-vs-deps**|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=dev16.0-preview2-vs-deps&jobname=Windows_VisualStudio_Integration_Tests&configuration=debug)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=dev16.0-preview2-vs-deps&view=logs)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=dev16.0-preview2-vs-deps&jobname=Windows_VisualStudio_Integration_Tests&configuration=release)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=dev16.0-preview2-vs-deps&view=logs)|
+**master**|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=master&jobname=Windows_VisualStudio_Integration_Tests&configuration=debug)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=245&branchname=master&view=logs)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=master&jobname=Windows_VisualStudio_Integration_Tests&configuration=release)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=245&branchname=master&view=logs)|
+**master-vs-deps**|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=master-vs-deps&jobname=Windows_VisualStudio_Integration_Tests&configuration=debug)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=245&branchname=master-vs-deps&view=logs)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=master-vs-deps&jobname=Windows_VisualStudio_Integration_Tests&configuration=release)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=245&branchname=master-vs-deps&view=logs)|
```

Comment:
😕 Now **master** works for me, but the rest of these are taking me to MachineLearning-CI

Reply:
The links are correct. The problem is we haven't enabled the integration tests in the other branches yet (why they're marked as never run). The ADO behavior for such a link is to take you to the first job definition vs. specified job definition.